src/createServer.js

```diff
@@ -1,3 +1,67 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+
+const {
+  convertToCase,
+} = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const errors = [];
+    const textToConvert = normalizedURL.pathname.slice(1);
+    const caseName = normalizedURL.searchParams.get('toCase');
+    const caseNames = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    if (!textToConvert) {
+      errors.push({
+        message: 'Text to convert is required.' +
+          ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!caseName) {
+      errors.push({
+        message: '"toCase" query param is required.' +
+          ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    } else if (!caseNames.includes(caseName)) {
+      errors.push({
+        message: 'This case is not supported.' +
+          ' Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+      });
+    }
+
+    if (errors.length) {
+      res.writeHead(400, 'Bad request', {
```

Comment:
also for status.
Tip* -  you can create object ( in ts - enum) - STATUS_CODES or something like that)